src/client/src/assets/i-hamburger.svg

```diff
@@ -0,0 +1,5 @@
+<svg alt="expand menu" width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M20 0H0V0.938889H20V0Z" fill="white"/>
+<path d="M20 7.50391H0V8.44279H20V7.50391Z" fill="white"/>
+<path d="M20 14.5391H0V15.478H20V14.5391Z" fill="white"/>
+</svg>
```

Comment:
Is this icon used somewhere? I don't see a reference. In my sidebar PR I kinda hardcoded hamburger without svg icon, so this could be used instead ;)